upstream-community-operators/azure-service-operator/0.37.0/azure-service-operator.v0.37.0.clusterserviceversion.yaml

```diff
@@ -1227,11 +1227,11 @@ spec:
           name: azureoperatorsettings
           namespace: operators
         data:
```

Comment:
This should be "stringData`.  'data' assumes the values are already base64 encoded.